client/js/mousehandling.js

```diff
@@ -43,7 +43,7 @@ function inputHandler(name, e) {
       const ms = mouseStatus[target.id];
       const timeSinceStart = +new Date() - ms.start;
       const pixelsMoved = ms.coords ? Math.abs(ms.coords[0] - ms.downCoords[0]) + Math.abs(ms.coords[1] - ms.downCoords[1]) : 0;
-      if(ms.status != 'initial' && (jeEnabled || ms.widget.p(edit ? 'movableInEdit' : 'movable')))
+      if(ms.status != 'initial' && (ms.widget.p(edit ? 'movableInEdit' : 'movable')))
```

Comment:
Parenthesis no longer needed but doesn't matter.

Reply:
Drag and drop still works as normal when the JSON editor is open.

Reply:
But there's no way to move for example holders with the mouse now. I don't think they should just be movable like in the normal edit mode but probably when holding `CTRL` or something.

But maybe it's fine that you can toggle the normal edit mode inside the JSON editor.

Reply:
> But maybe it's fine that you can toggle the normal edit mode inside the JSON editor.

That was my thought, though I do like the idea of a keyboard modifier as well.

Reply:
We could also make the test `widget.p((edit || jeEnabled) ? 'movableInEdit' : 'movable' ))` so that the `movableInEdit` property governs in the JSON editor regardless of if the room area is in play mode or edit mode.

Reply:
Yeah, but I don't think I would prefer that.